rpc/connector/connector.proto

```diff
@@ -35,6 +35,7 @@ service Connector {
   // Adds an intercept to a workload.  Requires having already called
   // Connect.
   rpc CreateIntercept(CreateInterceptRequest) returns (InterceptResult);
+  rpc CreatePoddIntercept(CreateInterceptRequest) returns (InterceptResult);
```

Comment:
Why is this method called `CreatePoddIntercept` and what differs it from a `CreateIntercept`. Is `Podd` really a term that we want to expose in our APIs?